pysted/base.py

```diff
@@ -964,12 +968,16 @@ def get_effective(self, datamap_pixelsize, p_ex, p_sted):
         # time averaged power emitted per molecule if there was no deexcitation
         # caused by the depletion beam
         sigma_abs = self.fluo.get_sigma_abs(self.excitation.lambda_)
-        excitation_probability = sigma_abs * i_ex * self.fluo.qy
+        excitation_probability = sigma_abs * i_ex * self.fluo.qy #without saturation
```

Comment:
I don't understand why this should be calculated, if the variable is overwritten under ...